src/main/java/upeldev/com/github/upel3/controllers/ActivityController.java

```diff
@@ -77,8 +77,10 @@ public String activity(Model model, Principal principal, HttpServletRequest requ
             double points = 0;
             List<Grade> grades;
             if(currentUser.getCoursesEnrolledIn().contains(course)){
-                grades = gradeService.findGradeByCourseAndUserAndActivity(activity.getCourse(), currentUser, activity);
-                if(!grades.isEmpty()) points = grades.get(grades.size() - 1).getValue();
+                //grades = gradeService.findGradeByCourseAndUserAndActivity(activity.getCourse(), currentUser, activity);
+                grades = activityService.getStudentGradesInActivity(activity, currentUser);
+                if(!grades.isEmpty())
+                    points = grades.get(grades.size() - 1).getValue(); //currently uses value of random grade (if there is both UserGrade and GroupGrade)
```

Comment:
I think there is no point in having the possibility of grading one student in one activity both as a part of group and individually. For now it's ok, but we need to take care of that in the next sprint.